DESIGN.md

```diff
@@ -4,8 +4,8 @@
 ## Summary
 
 Redalert is a tool used for system validation, similar to ChefSpec/ServerSpec.
-It is written in Go and configured with YAML to be more familiar to those
-accustomed to working with Ansible.
+It is written in Go and configured with YAML and should be familiar to those
+accustomed to working with similarly modeled tools such as Ansible.
```

Comment:
Remove "similarly modeled tools such as" and this lgtm

Reply:
ah, sgtm too